pinot-core/src/main/java/org/apache/pinot/core/query/reduce/ExecutionStatsAggregator.java

```diff
@@ -338,13 +357,19 @@ public void setStats(@Nullable String rawTableName, BrokerResponseNative brokerR
   }
 
   public void setStageLevelStats(@Nullable String rawTableName, BrokerResponseStats brokerResponseStats,
-      @Nullable BrokerMetrics brokerMetrics) {
+      @Nullable BrokerMetrics brokerMetrics, boolean traceEnabled) {
```

Comment:
No need to pass this flag imo. It is already part of the constructor.

Reply:
good catch!